__tests__/__assets__/cap-proj/scp-deployment/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
To create all the files needed for SAP CP Cloud Foundry deployment we could use [cds-dbm build](https://github.com/mikezaschka/cds-dbm#build)

Reply:
removed the entire folder as it is not used for deployment
rather, cds deploy (via cds-pg's simple deploy mechanism) is used (from the -srv module) as we don't need delta deployments here